corehq/apps/data_cleaning/models/session.py

```diff
@@ -14,7 +14,7 @@
 BULK_OPERATION_CHUNK_SIZE = 1000
 MAX_RECORDED_LIMIT = 100000
 MAX_SESSION_CHANGES = 200
-APPLY_CHANGES_WAIT_TIME = 15  # seconds
+APPLY_CHANGES_WAIT_TIME = 3  # seconds
```

Comment:
This is the artificial buffer right? Is 3 seconds enough 😬?

Reply:
is it? I had to argue with @kcowger we need more, but he felt 15 was too much. but maybe someone from platform will offer better advice?

Reply:
If we ultimately tie this to pillow lag then maybe it isn't too big of a deal? But I look at a graph like [this](https://app.datadoghq.com/dashboard/ewu-jyr-udt/change-feeds-pillows?fromUser=false&fullscreen_end_ts=1747941117180&fullscreen_paused=false&fullscreen_refresh_mode=sliding&fullscreen_section=overview&fullscreen_start_ts=1747937517180&fullscreen_widget=210889790&refresh_mode=sliding&tpl_var_pillow%5B0%5D=case-pillow&from_ts=1747937504796&to_ts=1747941104796&live=true) which makes 3 seconds feel tough. Granted that graph is the max lag time so it paints the worst picture.